csrc/host_ir/host_ir.h

```diff
@@ -492,6 +496,8 @@ class ForLoop : public Expr {
 
   NVFUSER_DECLARE_CLONE_AND_CREATE
 
+  static ForLoop* createFromIterDomain(Val* index, IterDomain* iter_domain);
```

Comment:
I'm on the fence about this. The method is coupled with the ForLoop class so I moved here to save some typing. The downside is less access control because createFromIterDomain could access private fields/methods of ForLoop.